### ISSUE TITLE
Project licensing clarifications

### ISSUE THREAD

**Issue by raducoravu** [w3c/epubcheck#925]
Oxygen XML Editor bundles inside the EPUBCheck project and our legal department has some licensing questions/remarks:

1) Who is the actual owner of project?

    is it "International Digital Publishing Forum" as specified in the pom.xml file?
    is it "Adobe System Incorporated" as specified in https://github.com/w3c/epubcheck/blob/master/COPYING.txt file?

    Please update the copyright statement accordingly.

2) What is the actual license of the project?

    is it MIT as specified in the COPYING.txt file on the GitHub repository? (https://github.com/w3c/epubcheck/blob/master/COPYING.txt)
    is it "New BSD" as specify by the maven deployment file (https://github.com/w3c/epubcheck/blob/master/pom.xml) and in the README file? (see the bottom of the page:
    https://github.com/w3c/epubcheck "EPUBCheck is made available under the terms of the New BSD License")

    In any case, please clarify the presence of Apache 2.0/ MPL-1.0 licenses in the https://github.com/w3c/epubcheck/tree/master/src/main/licenses directory.
    Please let us know the reason to include such licenses and pay attention to the way open source licenses can be combined (see more details on this page: https://dwheeler.com/essays/floss-license-slide.html)

**Comment by rdeltour** [comment]
> 1. Who is the actual owner of project?
>    Please update the copyright statement accordingly.

The project is now owned by W3C.
I did in fact raise the copyright statement issue to the steering committee a couple weeks ago, and we’re waiting for their guidance (along with W3C's legal team) to update the copyright notice. In any case, per the license, the previous copyright statement (from Adobe) will be kept; new statements will come in addition.

> 2. What is the actual license of the project?

This is an important question indeed (😅), and coincidently I noticed this confusing situation very recently as well; this has also been brought up to W3C's digital publishing steering committee, as well as the W3C legal team.

This confusion appears to be very old, since [the old Google code project](https://code.google.com/archive/p/epubcheck/) already stated it was licensed under New BSD, that's why this info was [copied in the readme file](https://github.com/w3c/epubcheck/commit/deaf0d8d) and POM file when importing the project to GitHub years ago. But your correct that the license file is indeed MIT, so I suppose that effectively makes the project licensed under MIT (contrary to what has been said in the readme for a long while).

We're waiting from the feedback of W3C's legal team, and we'll clarify the situation ASAP.
